ui/static/app/record/record.component.ts

```diff
@@ -64,21 +64,27 @@ export class RecordComponent extends SearchComponent implements OnInit {
     this.additionalRouteParams["showAll"] = this.routeParams.get("showAll");
     let url_offset: string = this.routeParams.get("offset");
     this.currentOffset = url_offset ? Number(url_offset) : 0;
-    this.searchValue = this.routeParams.get("search");
+    let search: string = this.routeParams.get("search");
+    this.searchValue = (search !== "null") ? search : "";
```

Comment:
when is the case when null will be passed as a string? cannot we pass empty string as a query string then?